tests/pkg_tweetnacl/tests/01-run.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2016 Francisco Acosta <francisco.acosta@inria.fr>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+def testfunc(child):
+    child.expect('Creating keypair ALICE...\r\n')
+    child.expect('Done.\r\n')
+    child.expect('Creating keypair BOB...\r\n')
+    child.expect('Done.\r\n')
+    child.expect('Encrypting using pk_bob...\r\n')
+    child.expect('Decrypting...\r\n')
+    child.expect('Decryption successful. content: "0123456789abcdef"\r\n')
+    child.expect('    pid | state    Q | pri \r\n')
```

Comment:
Uhm... yeah we can remove it. However I can maybe put some more information about this in a README.md, what do you think?